macros/variables.cfg

```diff
@@ -8,16 +8,46 @@ variable_z_drop_speed: 32
 variable_brush_clean_speed: 130
 
 # Physical Z endstop position
-variable_z_endstop_x: 205.3
-variable_z_endstop_y: 305.5
+variable_z_endstop_x: -1
+variable_z_endstop_y: -1
```

Comment:
Now that we use the [z_calibration] config, we can remove those two lines I think to avoid confusion.

Also, maybe we will need to think about a way to do it if the z_calibration plugin is not installed... Bu this can be done later.

Reply:
Line removed